### DISCUSSION TITLE
Requested Irish language on Weblate

### DISCUSSION THREAD

**Discussion by aindriu80** [searxng/searxng#3878] [Q&A] [answered]
Hi there,

A while ago, I requested the addition of Irish (Gaeilge, language code: ga) to SearxNG, but I haven't heard anything of it. I understand there might have been a technical issue or perhaps the request was missed. Could someone kindly look into adding Irish as a supported language on Weblate?

Thanks for your time and all the great work on SearxNG!

Kind regards,
Aindriú

**Comment by return42** [comment, marked as answer]
Continued in https://github.com/searxng/searxng/pull/4125#issuecomment-2559260490
